generate_vcard_qr_code.py

```diff
@@ -1,41 +1,75 @@
-#!/usr/bin/env python3
-
 import qrcode
 
 
+# QR Code settings
+QR_VERSION = 1
+QR_ERROR_CORRECTION = qrcode.constants.ERROR_CORRECT_L
+QR_BOX_SIZE = 10
+QR_BORDER = 4
+
 def get_vcard_data():
     print("Please enter the following information for the vCard:")
+    
+    # Input collection with basic validation
+    first_name = input("First Name: ").strip()
+    last_name = input("Last Name: ").strip()
+    organization = input("Organization: ").strip()
+    url = input("URL: ").strip()
+    email = input("Email: ").strip()
+    phone = input("Phone: ").strip()
+    street = input("Street + number: ").strip()
+    city = input("City: ").strip()
+    postal_code = input("Postal Code: ").strip()
+    country = input("Country: ").strip()
```

Comment:
The optional properties should be marked as optional in the user prompt. Alternatively (or both) the mandatory fields should be marked as such.

Reply:
Made these optional.